test/parallel/test-tls-async-cb-after-socket-end.js

```diff
@@ -1,66 +1,67 @@
 'use strict';
-
 const common = require('../common');
 if (!common.hasCrypto)
   common.skip('missing crypto');
 const fixtures = require('../common/fixtures');
 const SSL_OP_NO_TICKET = require('crypto').constants.SSL_OP_NO_TICKET;
 const tls = require('tls');
 
+// Check tls async callback after socket ends
+
 const options = {
   secureOptions: SSL_OP_NO_TICKET,
   key: fixtures.readSync('test_key.pem'),
   cert: fixtures.readSync('test_cert.pem')
 };
 
-const server = tls.createServer(options, function(c) {
+const server = tls.createServer(options, (c) => {
 });
```

Comment:
Just so it is not overlooked: https://github.com/nodejs/node/pull/18985#discussion_r170582654

Reply:
fixing it

Reply:
fixed :) Thanks for pointing out